napari/layers/base/base.py

```diff
@@ -710,22 +705,6 @@ def ndim(self):
         """int: Number of dimensions in the data."""
         return self._ndim
 
-    @property
-    def selected(self):
```

Comment:
Given that we have the `_on_selection` callback right now, I might just deprecate these for now otherwise it will be a big breaking change. Could even keep event emission and temporarily wire up callbacks to update selection